sw/host/opentitanlib/src/app/config/hyperdebug_cw340.json

```diff
@@ -48,6 +48,12 @@
       "mode": "Alternate",
       "pull_mode": "PullUp",
       "alias_of": "IOA7"
+    },
+    {
+      "name": "IOA2",
+      "mode": "OpenDrain",
+      "level": false,
+      "pull_mode": "PullDown"
```

Comment:
Open drain and pulldown is an odd combination. Either it will be strongly driven low, or weakly pulled low. Either way the OT chip will read a low level.  Was the intention to set pull-up? That is the usual choice with open drain.

Reply:
The desire is to have the FPGA see a level-low on IOA2 by default.  This customer is going to configure IOA2-high as the trigger for rescue and they normally set the pull down in pinmux.  However, the FPGA doesn't model pull resistors in the pads, so we need to present a level-low to prevent rescue from triggering by default in their tests.

I'm agnostic about whether its PushPull+Low or OpenDrain+PullDown.